index.js

```diff
@@ -166,6 +167,15 @@ function start() { // eslint-disable-line max-statements
     // Set the X-XSS-Protection HTTP header as a basic protection against XSS
     app.use(helmet.xssFilter());
 
+    /*
+     * Allow routes, that need to be accessed from scripting queries, access
+     * throught the cross-origin resource sharing HTTP standard e.g. Bots that
+     * needs to get needs to get current botData to continue its action
+     */
+    if (featureToggles.isFeatureEnabled('enableCORS')) {
+      conf.corsRoutes.forEach((rte) => app.use(rte, cors()));
```

Comment:
@iamigo @tausifmuzaffar  wondering if we need the "enableCORS" flag? The presence or absence of something in the corsRoutes can tell us if CORS is enabled or not. Any thoughts?